File: src/zadania/zadanie 1/main.rb
print "> "
n = gets.chomp().to_i()

parzyste, nieparzyste = [], []

n.times do |i|
    print "> "
    num = gets.chomp().to_i()

    if num % 2 == 0
        parzyste << num
    else 
        nieparzyste << num
    end
end

if parzyste.length > nieparzyste.length
    puts "P"
elsif parzyste.length < nieparzyste.length
    puts "NP"
else
    puts "="
end

File: src/zadania/zadanie 1/main.py
n = int(input("> "))

parzyste, nieparzyste = [], []

for i in range(n):
    num = int(input("> "))
    if num % 2 == 0:
        parzyste.append(num)
    else:
        nieparzyste.append(num)


query = "P" if len(parzyste) > len(nieparzyste) else "NP" if len(parzyste) < len(nieparzyste) else "="
print(query)